src/components/Form/DropdownLabel.tsx

```diff
@@ -71,7 +71,24 @@ export const Label = ({
   children,
   Element = 'span',
   field,
-  ...props
+  value,
+  style,
+  onChange,
+  onFocus,
+  onBlur
+}: {
+  top?: boolean
+  focus?: boolean
+  error?: boolean
+  text?: string
+  children: React.ReactNode | React.ReactNode[]
+  Element?: string
+  field?: boolean
+  value?: string
+  style?: { [key: string]: string }
```

Comment:
I believe this is a good use case for `Record<string, string>`

Reply:
Is it just me or is `{ [key: string]: string }` just infinitely more readable than `Record<string, string>`? Like, if I know nothing of Typescript, I still probably get the former, the latter... i don't know what it truly means :D

Reply:
but done, because why not ^^